tests/test_cofe/test_clusterspace.py

```diff
@@ -380,20 +369,21 @@ def get_corr(occu, space, matrix):
     # Symmetrically equivalent matrices should give the same correlation
     # vectors on the same structure, when using the default orbit mapping.
     cluster_subspace._supercell_orb_inds = {}
-    for s, c in zip(structures, corrs):
+    for structure, expected in zip(structures, corrs):
+        predicted = cluster_subspace.corr_from_structure(structure)
```

Comment:
Why remove supercell matrix here?

Reply:
the different supercell matrices are passed below. We can remove the one where no sc matrix is passed and that will prevent possible errors from structures that do not match